app/pages/editorg/template.test.js

```diff
@@ -113,10 +113,10 @@ describe('editorg page', () => {
         const checkboxInput = $('[data-test-id="catalogue-agreement-checkbox"] input');
         expect(checkboxInput.length).toEqual(1);
         expect(checkboxInput.attr('id')).toEqual('catalogue-agreement-checkbox');
-        expect(checkboxInput.attr('name')).toEqual('catalogueagreementsigned');
+        expect(checkboxInput.attr('name')).toEqual('catalogueAgreementSigned');
         expect(checkboxInput.attr('type')).toEqual('checkbox');
         expect(checkboxInput.attr('checked')).toEqual(undefined);
-        expect(checkboxInput.attr('value')).toEqual('false');
+        expect(checkboxInput.attr('value')).toEqual('catalogueAgreementSigned');
```

Comment:
ok so when the box is checked. The value of it being checked is `catalogueAgreementSigned`. Like with a dropdown box. There is a value associated to what is displayed?